### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
Cleaning-up

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: scripts/upgrade [hunk 1]
```diff
@@ -21,7 +21,7 @@ no_sso=$(ynh_app_setting_get --app=$app --key=no_sso)
 server_password=$(ynh_app_setting_get --app=$app --key=server_password)
 su_passwd=$(ynh_app_setting_get --app=$app --key=su_passwd)
 welcometext=$(ynh_app_setting_get --app=$app --key=welcometext)
-registerName=$(ynh_app_setting_get --app=$app --key=registerName)
+registername=$(ynh_app_setting_get --app=$app --key=registername)
 port=$(ynh_app_setting_get --app=$app --key=port)
```
Comment (alexAubin) [inline scripts/upgrade:25]: Zblerg I guess we need to add somewhere the small migration snippet:

```bash
if [ -z "$registername" ];
then
      	registername=$(ynh_app_setting_get "$app" registerName)
      	ynh_app_setting_set "$app" registername "$registername"
      	ynh_app_setting_delete "$app" registerName
fi
```

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Review by alexAubin — APPROVED** [review]
:+1:

**Pull request merged**